tensorflow/python/keras/losses.py

```diff
@@ -646,8 +646,7 @@ class CategoricalHinge(LossFunctionWrapper):
   """Computes the categorical hinge loss between `y_true` and `y_pred`.
 
   `loss = maximum(neg - pos + 1, 0)`
-  where `neg = sum(y_true * y_pred)` and `pos = maximum(1 - y_true)`
-
```

Comment:
This change should not be in this PR

Reply:
I am sorry. I am unable to remove this commit as this was in the same branch.

Reply:
I am Closing this PR and creating a fresh PR